eng/pipelines/steps/PublishOpenDebugAD7.yml

```diff
@@ -0,0 +1,41 @@
+parameters:
+  Configuration: 'Lab.Release'
+  RuntimeID: 'win-x64'
+  SignedBinariesFolder: ''
+
+steps:
+- script: |
+    dotnet publish $(Build.SourcesDirectory)\src\OpenDebugAD7\OpenDebugAD7.csproj -c ${{ parameters.Configuration }} -r ${{ parameters.RuntimeID }} --self-contained -o $(Build.StagingDirectory)\debugAdapters\${{ parameters.RuntimeID }}
+    copy ${{ parameters.SignedBinariesFolder }}\Release\vscode\OpenDebugAD7.dll "$(Build.StagingDirectory)\debugAdapters\${{ parameters.RuntimeID }}\.
+    copy ${{ parameters.SignedBinariesFolder }}\Release\vscode\Microsoft.DebugEngineHost.dll $(Build.StagingDirectory)\debugAdapters\${{ parameters.RuntimeID }}\.
+    copy ${{ parameters.SignedBinariesFolder }}\Release\Microsoft.MIDebugEngine.dll $(Build.StagingDirectory)\debugAdapters\${{ parameters.RuntimeID }}\.
+    copy ${{ parameters.SignedBinariesFolder }}\Release\Microsoft.MICore.dll $(Build.StagingDirectory)\debugAdapters\${{ parameters.RuntimeID }}\.
+  displayName: "Publish OpenDebugAD7 ${{ parameters.RuntimeID }}"
+
+- ${{ if startsWith(parameters.RuntimeID, 'win-') }}:
+  - script: |
+      copy ${{ parameters.SignedBinariesFolder }}\Release\vscode\OpenDebugAD7.exe $(Build.StagingDirectory)\debugAdapters\${{ parameters.RuntimeID }}\.
+    displayName: "Copy OpenDebugAD7.exe"
+
+- template: ../tasks/SignVerify.yml
```

Comment:
Should this only run for `win-` runtime ids?

Reply:
We do have plenty of '.dll' files for all runtimes that should be Authenticode signed...